src/main/java/com/dev/cinema/util/HibernateUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.dev.cinema.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new RuntimeException("Error creating Session Factory");
```

Comment:
Same here, maybe custom?

Reply:
this exception has a different nature, so we should to throw RuntimeException here

Reply:
Agree. you are right.